src/i18n/messages-fr.po

```diff
@@ -0,0 +1,875 @@
+msgstr " est passé en ligne."
+
+msgid "You share file:"
+msgstr "Vous partagez le fichier:"
+
+msgid "Incoming file:"
+msgstr "Fichier entrant:"
+
+msgid "You shared your location:"
+msgstr "Vous partagez votre emplacement:"
+
+msgid "Location received:"
+msgstr "Emplacement reçu:"
+
+msgid "You accepted the contact request."
+msgstr "Vous avez accepté la demande de mise en relation."
+
+msgid "You rejected the contact request."
+msgstr "Vous avez refusé la demande de mise en relation."
+
+msgid "You sent a contact request."
+msgstr "Vous avez envoyé une demande de mise en relation"
+
+msgid "Your contact request was accepted."
+msgstr "Votre requête de mise en relation a été acceptée."
+
+msgid "Incoming contact request."
+msgstr "arrivée d'une demande de mise en relation"
+
+msgid "Your contact request was rejected."
+msgstr "Votre requête de mise en relation a été refusée."
+
+msgid "Edit Contact"
+msgstr "Modifier le contact"
+
+msgid "Your browser does not support WebRTC. No calls possible."
+msgstr "Votre navigateur ne supporte pas WebRTC. Appels impossible"
+
+msgid "Close this window and disconnect?"
+msgstr "Fermer cette fenêtre et se déconnecter ?"
+
+msgid "Contacts Manager"
+msgstr "Gestionnaire de contacts"
+
+msgid "Restart required to apply updates. Click ok to restart now."
+msgstr ""
+"Un redémarrage est requis pour appliquer les mises à jour. Cliquez"
+" pour redémarrer"
+
+msgid "Failed to access camera/microphone."
+msgstr "IMpossible d'accéder à la webcam et au micro."
+
+msgid "Failed to establish peer connection."
+msgstr "IMpossible d'établir la connexion de pair a pair."
+
+msgid "We are sorry but something went wrong. Boo boo."
+msgstr "Nous sommes désolé, mais quelque chose à planté. Ouin"
+
+msgid "Oops"
+msgstr "Oups"
+
+msgid "Peer connection failed. Check your settings."
+msgstr "La connection avec le pair à échoué. Vérifié vos réglages."
+
+msgid "User hung up because of error."
+msgstr "Le participant à raccroché suite à une erreur"
+
+msgid " is busy. Try again later."
+msgstr " est occupé. Réessayez plus tard."
+
+msgid " rejected your call."
+msgstr " à refuser votre appel."
+
+msgid " does not pick up."
+msgstr " n'a pas répondu"
+
+msgid " tried to call you"
+msgstr " a essayé de vous appeler"
+
+msgid " called you"
+msgstr " vous a appelé"
+
+msgid "Your browser is not supported. Please upgrade to a current version."
+msgstr ""
+"Votre navigateur n'est pas supporté. MErci de le mettre à jout vers une"
+"version plus récente"
+
+msgid "Chat with"
+msgstr "Chat avec"
+
+msgid "Message from "
+msgstr "Massage de  "
+
+#, python-format
+msgid "You are now in room %s ..."
+msgstr "Vous avez rejoint la salle  %s ..."
+
+msgid "Your browser does not support file transfer."
+msgstr "Votre navigateur ne supporte pas les transferts de fichier."
+
+msgid "Could not load PDF: Please make sure to select a PDF document."
+msgstr ""
+"Impossible de charger le PDF : merci de vérifier que vous avez "
```

Comment:
Unnecessary blank before `:`. Also above you have `pdf`all lowercase, here it's all uppercase - please use the same style everywhere.